pkg/controllers/csi/config.go

```diff
@@ -31,6 +31,7 @@ const (
 	OverlayWorkDirPath   = "work"
 	SharedAgentBinDir    = "codemodules"
 	SharedAppMountsDir   = "appmounts"
+	SharedDynaKubesDir   = "dyna.kubes"
```

Comment:
How about `shared-dk` or `shared-dynakube` or something along those lines?

I mean I am not completely against this name but just wanted to give some ideas

Reply:
you can create a DynaKube with name `shared-dk` and `shared-dynakube`

[the validation will not catch it](https://github.com/Dynatrace/dynatrace-operator/blob/1ade90a1a28f3216c48420f3c1c48fc97e48ce69/pkg/api/validation/dynakube/dynakube_name.go#L20)

Reply:
fair enough 😅 I guess `dyna.kubes` is the best possible way

Reply:
what about `_dynakubes` or `_shared`?